src/components/Nav/RightNav.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import styled from "styled-components";
+import Link from "next/link";
+
+export const RightNav = ({ open }) => {
+  return (
+    <div>
+      <Ul open={open}>
+        <li>
+          <Link href="/">
+            <a>Home</a>
+          </Link>
+        </li>
+        <li>
+          <Link href="/configurator">
+            <a>Configurator</a>
+          </Link>
+        </li>
+        <li>
+          <Link href="/offer">
+            <a>Übersicht</a>
+          </Link>
+        </li>
+        <li>
+          <Link href="/export">
+            <a>Export</a>
+          </Link>
+        </li>
+      </Ul>
+    </div>
+  );
+};
+
+const Ul = styled.ul`
+  list-style: none;
+  display: flex;
+  flex-flow: row nowrap;
+  color: #ff7b00;
```

Comment:
suggestion: use css variables for colors. you can place them in global styles like this :root { --bg-color-nav: #ff7b00} and simply call the var in here with color: var(--bg-color-nav).
this can be rly helpfull as the project grows